tests/cross/src/test/scala/tests/pc/CompletionMillIvySuite.scala

```diff
@@ -17,6 +17,16 @@ class CompletionMillIvySuite extends BaseCompletionSuite {
     filename = "build.sc"
   )
 
+  check(
+    "source",
+    """|val dependency = ivy"io.cir@@"
```

Comment:
I think we have a lot of places where we check if something is a scala file:
- https://github.com/scalameta/metals/blob/f3657def9a69b77a43e9cef8d5f51273c005674b/mtags/src/main/scala/scala/meta/internal/mtags/ScalametaCommonEnrichments.scala#L175
- https://github.com/scalameta/metals/blob/f3657def9a69b77a43e9cef8d5f51273c005674b/tests/unit/src/main/scala/tests/InputFile.scala#L21
- https://github.com/scalameta/metals/blob/f3657def9a69b77a43e9cef8d5f51273c005674b/mtags-shared/src/main/scala/scala/meta/internal/mtags/CommonMtagsEnrichments.scala#L252

Reply:
Do you need help figuring why the test is failing? I would just try to change those places mentioned above (refactor would probably be prudent at some point, sorry about that) and see if the tests starts passing.

Reply:
Ach, right, so I guess there is another spot I forgot about :thinking: